include/aspect/geometry_model/ellipsoidal_chunk.h

```diff
@@ -74,23 +71,12 @@ namespace aspect
             Point<3>
             push_forward(const Point<3> &chart_point) const;
 
-            Point<dim>
-            rotate_rectangle_to_parallelogram(const Point<dim> &x) const;
-
-            Point<dim>
-            rotate_parallelogram(const Point<dim> &x) const;
-
-
-
           private:
             double semi_major_axis_a;
             double eccentricity;
             double semi_minor_axis_b;
-            double rot_para_to_para_angle;
-            double para_to_rect_angle;
-            double rotation_longitude;
-            double rotation_latitude;
             double bottom_depth;
+            std::vector<std::vector<double> > corners;
```

Comment:
I think it would be useful to document what the elements of this array correspond to. I suspect the coordinates of the corners of the model, but in which coordinate system? And how many elements is this array supposed to have? Also, would it be less confusing if you stored a `std::vector<Point<dim>>` here? This would make the semantics of the variable clearer.